psl/psl/tests/capabilities/sqlite.rs

```diff
@@ -108,35 +108,6 @@ fn unique_index_names_support() {
     expectation.assert_eq(&error);
 }
 
-#[test]
-fn json_support() {
-    let dml = indoc! {r#"
-        datasource db {
-          provider = "sqlite"
-          url = "file:test.db"
-        }
-
-        model User {
-          id   Int @id
-          data Json
-        }
-    "#};
-
-    let error = parse_unwrap_err(dml);
-
-    let expectation = expect![[r#"
-        [1;91merror[0m: [1mError validating field `data` in model `User`: Field `data` in model `User` can't be of type Json. The current connector does not support the Json type.[0m
-          [1;94m-->[0m  [4mschema.prisma:8[0m
-        [1;94m   | [0m
-        [1;94m 7 | [0m  id   Int @id
-        [1;94m 8 | [0m  [1;91mdata Json[0m
-        [1;94m 9 | [0m}
-        [1;94m   | [0m
-    "#]];
-
-    expectation.assert_eq(&error);
-}
```

Comment:
Why did you delete this? Just adapt the expectations instead

Reply:
all expectations in the file assert for errors, but I can add it back asserting for no error